_data/team.yml

```diff
@@ -0,0 +1,190 @@
+# <p class="label">🚧 Safety 🚧</p>
+# <p class="label">⚙️ Member ⚙️</p>
+# <p class="label">⭐️ Team Captain ⭐️</p>
+# <h5>Name 'year</h5>
+# <p>Name is a first year subject.</p>
+
+team:
+  # Leads
+  - name: "Mya Taheri"
+    year: "'25"
+    role: "⭐️ Team Captain ⭐️"
+    image: "/images/team/mya-taheri.png"
+    bio: "Mya is a fourth-year programmer and team captain."
+
+  - name: "Ian Beaupain"
+    year: "'25"
+    role: "🔧 Mechanic Lead 🔧"
+    image: "/images/team/ian-beaupain.png"
+    bio: "Ian is a fourth-year mechanic."
+
+  - name: "Ben Berol"
+    year: "'25"
+    role: "🖥️ Programming Lead 🖥️"
+    image: "/images/team/ben-berol.png"
+    bio: "Ben is a fourth-year programmer."
+
+  - name: "Cameron Dunn"
+    year: "'25"
+    role: "🖥️ Programming Lead 🖥️"
+    image: "/images/team/cameron-dunn.png"
+    bio: "Cameron is a fourth-year programmer."
+
+  - name: "Nina Miller"
+    year: "'25"
+    role: "🔌 Electrician Lead 🔌"
+    image: "/images/team/nina-miller.png"
+    bio: "Nina is a fourth-year electrician."
+
+  # Programmers
+  - name: "CJ Artz"
+    year: "'26"
+    role: "🖥️ Programmer 🖥️"
+    image: "/images/team/cj-artz.png"
+    bio: "CJ is a second-year programmer."
+
+  - name: "Hannah Au"
+    year: "'26"
+    role: "🖥️ Programmer 🖥️"
+    image: "/images/team/hannah-au.png"
+    bio: "Hannah is a second-year programmer."
+
+  - name: "Oliver Hardi"
+    year: "'26"
+    role: "🖥️ Programmer 🖥️"
+    image: "/images/team/oliver-hardi.png"
+    bio: "Oliver is a second-year programmer."
+
+  - name: "Avery Hyra"
+    year: "'26"
+    role: "🖥️ Programmer 🖥️"
+    image: "/images/team/placeholder.png"
+    bio: "Avery is a first-year programmer."
+
+  - name: "Will Sucher"
+    year: "'28"
+    role: "🖥️ Programmer 🖥️"
+    image: "/images/team/placeholder.png"
+    bio: "Will is a first-year programmer."
+
+  # Mechanics
+  - name: "Zane Qumseya"
+    year: "'26"
+    role: "🔧 Mechanic 🔧"
+    image: "/images/team/placeholder.png"
+    bio: "Zane is a third-year mechanic."
+
+  - name: "Rihanna Fajutrao"
+    year: "'26"
+    role: "🔧 Mechanic 🔧"
+    image: "/images/team/rihanna-fajutrao.png"
+    bio: "Rihanna is a second-year mechanic."
+
+  - name: "Oliver Csaszi"
+    year: "'25"
+    role: "🔧 Mechanic 🔧"
+    image: "/images/team/placeholder.png"
+    bio: "Oliver is a second-year mechanic."
+
+  - name: "Eleanor Johnson"
+    year: "'25"
+    role: "🔧 Mechanic 🔧"
+    image: "/images/team/placeholder.png"
+    bio: "Eleanor is a second-year mechanic."
+
+  - name: "Oles Solohub"
+    year: "'27"
+    role: "🔧 Mechanic 🔧"
+    image: "/images/team/placeholder.png"
+    bio: "Oles is a second-year mechanic."
+
+  - name: "Max Sadler"
+    year: "'26"
+    role: "🔧 Mechanic 🔧"
+    image: "/images/team/placeholder.png"
+    bio: "Max is a first-year mechanic."
+
+  - name: "Tiberius Schmittel"
+    year: "'26"
+    role: "🔧 Mechanic 🔧"
+    image: "/images/team/placeholder.png"
```

Comment:
It occurs to me that you could just make the image property in the YML say either placeholder or the name since the /images/team/ and .png are the same for everyone and could be handled in the HTML. In fact you could even make it so that only "placeholder" needs to be specified and if there's no placeholder specified you just make the name lowercase and replace the spaces with hyphens. Would be much cleaner.